internal/manager/conditions_ingressapi_test.go

```diff
@@ -0,0 +1,148 @@
+package manager_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	extensionsv1beta1 "k8s.io/api/extensions/v1beta1"
+	netv1 "k8s.io/api/networking/v1"
+	netv1beta1 "k8s.io/api/networking/v1beta1"
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	"github.com/kong/kubernetes-ingress-controller/v2/internal/manager"
+)
+
+type ingressTestOpts struct {
+	enabled      bool
+	crdInstalled bool
+}
```

Comment:
If this is only used in the test below let's put this definition there so that it doesn't pollute scope of the whole package.

Reply:
Makes total sense. [f4584b3](https://github.com/Kong/kubernetes-ingress-controller/pull/3013/commits/f4584b309593c92827e9c3adea19e9f020765591)